hips/utils/healpix.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HEALpy wrapper functions.
+
+This module contains wrapper functions around HEALPix utilizing
+the healpy library
+"""
+
+__all__ = [
+    'boundaries',
+]
+
+import healpy
```

Comment:
There's an import convention of using `import healpy as hp`, see e.g. http://healpy.readthedocs.io/en/latest/tutorial.html

Let's always follow this convention and change to using it here.